infrastructure/variables.tf

```diff
@@ -128,8 +128,8 @@ variable "smtp_host" {
 }
 
 variable "reports_cron" {
-  default     = "0 0 18 * * *"
-  description = "Cron signature for job to send out reports to be executed. Default value is 6PM everyday"
+  default     = "0 0 18 ? * MON-FRI"
```

Comment:
Rather than having a default value ideal way would be default is null and prod.tfvars has that value. Job would be disabled on all other envrs anyways sp should not need the cron for non prod environments.

Reply:
I had it as no value initially :/